azkaban-common/src/main/java/azkaban/executor/mail/DefaultMailCreator.java

```diff
@@ -66,12 +66,12 @@ public boolean createFirstErrorMessage(ExecutableFlow flow,
     if (emailList != null && !emailList.isEmpty()) {
       message.addAllToAddress(emailList);
       message.setMimeType("text/html");
-      message.setSubject("Flow '" + flow.getFlowId() + "' has encountered a failure on "
-          + azkabanName);
+      message.setSubject("Flow '" + flow.getFlowId()
```

Comment:
I just followed the best practices I already saw in the file. Do you think it would be better to refactor the template using constant variables / string.format? What's the best practice now?